default/python/AI/ShipDesignAI.py

```diff
@@ -1581,18 +1581,18 @@ def _combat_rating(self):
         return combat_stats.get_rating(enemy_stats=self.additional_specifications.enemy)
 
 
-class WarShipDesigner(ShipDesigner):
+class MilitaryShipDesigner(ShipDesigner):
     basename = "Warship (do not build me)"
```

Comment:
Are you talking about line 1584?  For that either the current state of the PR or else MilitaryShipDesignerBaseClass would either be fine with me.  I was referring to the 'Warship' in line 1585, though (I am pretty sure that is the line I had selected when I made the comment, but I should have double checked how things appeared once I actually entered the comment)